src/vstest.console/CommandLine/CommandLineOptions.cs

```diff
@@ -282,13 +283,29 @@ internal Framework TargetFrameworkVersion
         /// <param name="source">Path to source file to look for tests in.</param>
         public void AddSource(string source)
         {
+            var extnList = new List<string>() { ".xap", ".appx", ".dll", ".exe" };
```

Comment:
This won't work

Reply:
Thank you for your feedback @singhsarab .
I will check further for a better solution and reach you if any help needed.